bounties/npm/object-path/1/vulnerability.json

```diff
@@ -0,0 +1,54 @@
+{
+    "PackageVulnerabilityID": 1,
+    "DisclosureDate": "2020-09-14",
+    "AffectedVersionRange": "*",
+    "Summary": "Prototype Pollution",
+    "Contributor": {
+        "Discloser": "Asjidkalam",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "object-path",
+        "URL": "https://github.com/mariocasciaro/object-path",
+        "Downloads": "18M"
+    },
+    "CWEs": [
+        {
+            "ID": "CWE-400",
+            "Description": ""
+        }
+    ],
+    "CVSS": {
+        "Version": "3.1",
+        "AV": "N",
+        "AC": "H",
+        "PR": "N",
+        "UI": "N",
+        "S": "U",
+        "C": "L",
+        "I": "L",
+        "A": "L",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "5.6"
+    },
+    "CVEs": [
+        ""
+    ],
+    "Repository": {
+        "URL": "https://github.com/mariocasciaro/object-path",
+        "Codebase": [
+            "JavaScript"
+        ],
+        "Owner": "mariocasciaro",
+        "Name": "object-path",
+        "Forks": "65",
+        "Stars": "870"
+    },
+    "Permalinks": [
+        "https://github.com/mariocasciaro/object-path/blob/master/index.js#L96"
```

Comment:
Could you change with https://github.com/mariocasciaro/object-path/blob/master/index.js#L110 ?

Reply:
Thanks @Mik317 
I have resolved the issue.